app/assets/javascripts/past_teams.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  var $selectedYear = $('select#past_teams');
+
+  $selectedYear.on('change', function(){
+    var yearParam = "?year="+this.value;
+    var newUrl;
+    var removed = window.location.search;
+
+    if(removed.length>0){
```

Comment:
What it does: takes the current URL, removes the search params from it, and then appends new ones, e.g. `?year="2016"` to it.

It then changes the current URL which makes the browser do another get request with the new params.

Afaik @juuh42dias, you could shorten this to just:
```javascript
$(function() {
  var $selectedYear = $('select#past_teams');

  $selectedYear.on('change', function(){
    var yearParam = "?year="+this.value;
    window.location.search = yearParam;
  });
});
```
It should to exactly the same.